docs/integrations/tools/index.md

```diff
@@ -1,22 +1,26 @@
 # Custom Tools 
 
-This section contains documentation for custom tools created by the community to extend Prompt flow's capabilities for specific use cases. These tools are developed following the guide on [Creating and Using Tool Packages](../../how-to-guides/develop-a-tool/create-and-use-tool-package.md). They are not officially maintained or endorsed by the Prompt flow team. For questions or issues when using a tool, please use the support contact link in the table below.
+This section contains documentation for custom tools created by the community to extend prompt flow's capabilities for specific use cases. These tools are developed following the guide on [Creating and Using Tool Packages](../../how-to-guides/develop-a-tool/create-and-use-tool-package.md). They are not officially maintained or endorsed by the prompt flow team. For questions or issues when using a tool, please use the support contact in the table below.
 
 ## Tool Package Index 
 
 The table below provides an index of custom tool packages. The columns contain:
 
-- **Package Name:** The name of the tool package. Links to the package documentation.
-- **Description:** A short summary of what the tool package does.
-- **Owner:** The creator/maintainer of the tool package.
-- **Support Contact:** Link to contact for support and reporting new issues.
+- **Package Name:** The name of the tool package. Links to the PYPI package page. To install the custom tool package, please execute the command `pip install <package name>`.
+- **Tool Name:** Tools in the package. Links to the tool documentation.
+- **Description:** A short summary of what the tool does.
+- **Support Contact:** The Github account to contact for support and reporting new issues.
 
-| Package Name | Description | Owner | Support Contact |  
+| Package Name | Tool Name | Description | Support Contact |  
 |-|-|-|-|
-|||||
+|[promptflow-vectordb](https://pypi.org/project/promptflow-vectordb/)| [Vector DB Lookup](./vector_db_lookup_tool.md) | Search vector based query from existing Vector Database. | dans-msft, Adarsh-Ramanathan |
```

Comment:
support contact needs pratical raising tool issue path, we do not expect all issues coming to pf repo, right?
create issue link is the best, email is a must-have, better to be an team email, not personal. And I really do not think name is helpful to tool issue troubleshooting.